recipes/opencv/4.x/conanfile.py

```diff
@@ -44,7 +44,8 @@ class OpenCVConan(ConanFile):
         "neon": [True, False],
         "dnn": [True, False],
         "dnn_cuda": [True, False],
-        "detect_cpu_baseline": [True, False],
+        "cpu_baseline": "ANY",
+        "cpu_dispatch": "ANY",
```

Comment:
reading https://github.com/opencv/opencv/wiki/CPU-optimizations-build-options#customizing-cmake-options, it's a limited set of values, but they are comma-separated, so there are so many combinations. I think `ANY` is practical enough for now, recipe won't validate the values, but I believe OpenCV itself will fail on incorrect values.

Reply:
Yes, it's fragile, but it's kind of option which only someone familiarized with OpenCV will use, I don't think people do a mess with it so frequently.